<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossplotGridsInputDialog</class>
 <widget class="QDialog" name="CrossplotGridsInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X-axis:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbXType"/>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="cbXName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y-axis:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbYType"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cbYName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAttribute">
     <property name="title">
      <string>Attribute</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="cbDisplayAttribute">
        <property name="text">
         <string>Display Attribute</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Attribute:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbAttributeType">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="cbAttributeName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbHorizon">
     <property name="title">
      <string>Horizon</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cbUseHorizon">
        <property name="text">
         <string>Use a horizon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Horizon:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="cbHorizonName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CrossplotGridsInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CrossplotGridsInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDisplayAttribute</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbAttributeType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDisplayAttribute</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbAttributeName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbUseHorizon</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbHorizonName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
